jep/201/README.adoc

```diff
@@ -1,296 +0,0 @@
-= JEP-201: Jenkins Configuration as Code
```

Comment:
hmm, why does it delete the JEP entirely?

Reply:
I guess @bitwiseman did moved jep-201 REAME file into this new template file, not just the content, so from a git history pov the initial JEP has been removed.

Reply:
force-pushed to fix initial commit